pkg/nodejoiner/addnodes.go

```diff
@@ -43,11 +49,12 @@ func NewAddNodesCommand(directory string, kubeConfig string) error {
 	return err
 }
 
-func saveParams(directory, kubeConfig string) error {
+func saveParams(directory, kubeConfig string, generatePXE bool) error {
```

Comment:
The pxe param is valid only for the add-nodes command but not for the monitor one - but I don't think it's (yet) worth having a more complicated management of the param file.
Anyhow, just to keep a single file, what about using a `*bool` and leaving to `nil` for the monitor command, to highlight in a slightly stronger manner that it's not used (rather than using `false`)